### PR TITLE
Pia 2942 write migration guide

### PR DESCRIPTION
**Pull request by vizaknaidavid**

### CODE REVIEW

File: CaptureSDK/GiniCaptureSDK/Documentation/source/Migration to version 3.x.x.md [hunk 1]
```diff
@@ -193,3 +193,18 @@ You can show a bottom navigation bar by passing true to `GiniConfiguration.share
 your own by implementing the `HelpBottomNavigationBarAdapter` interface and passing it to `GiniConfiguration.shared.helpNavigationBarBottomAdapter`.
 
 You can find more details [here][https://developer.gini.net/gini-mobile-ios/GiniCaptureSDK/3.0.0-beta01/features.html#help-screen-customization] and [here](https://www.figma.com/file/H4WFJ4xaw4YNU4VaJYWiQq/iOS-Gini-Capture-SDK-2.0.0-UI-Customisation?node-id=40%3A584&t=LCqm8Cwfh9PGUM0E-1).
+
+## Analysis screen
+
+The new analysis screen uses the global UI customization options. You can discard the old screen specific
+customizations.
+
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Documentation/source/Migration to version 3.x.x.md:201]: [Suggestion] Could you please add the ### Breaking Changes section ( move the lines from 202 in it? and as well ### New Features ( add Custom loading indicator here)

**Review by zladzeyka — COMMENTED** [review]
Looks good. 😊 Check the migration for ## Help screens.

File: CaptureSDK/GiniCaptureSDK/Documentation/source/Migration to version 3.x.x.md [hunk 2]
```diff
@@ -193,3 +193,26 @@ You can show a bottom navigation bar by passing true to `GiniConfiguration.share
...
+The following string keys now represent suggestion titles with new keys added for describing the tips.
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Documentation/source/Migration to version 3.x.x.md:202]: [Suggestion] could you please move it to the breaking changes? 🙈

**Review by zladzeyka — COMMENTED** [review]
Thank you looks great! I have only 1 tiny wish 😊

File: CaptureSDK/GiniCaptureSDK/Documentation/source/Migration to version 3.x.x.md (hunk 2 above)
Comment (vizaknaidavid) [reply CaptureSDK/GiniCaptureSDK/Documentation/source/Migration to version 3.x.x.md:207]: I did not consider it. breaking change, that's why I left it there, but it is safe now 💪 😄

**Review by zladzeyka — APPROVED** [review]
Looks good 👏🏻

**Pull request merged**